src/commands.rs

```diff
@@ -0,0 +1,98 @@
+//! The command line interface for the simulation.
+use crate::log;
+use crate::model::Model;
+use crate::settings::Settings;
+use ::log::info;
+use anyhow::{Context, Result};
+use clap::{Parser, Subcommand};
+use include_dir::{include_dir, Dir};
+use std::path::PathBuf;
+
+/// The directory containing the example models.
+pub const EXAMPLES_DIR: Dir = include_dir!("examples");
+
+#[derive(Parser)]
+#[command(version, about)]
+/// The command line interface for the simulation.
+pub struct Cli {
+    #[command(subcommand)]
+    /// The available commands.
+    pub command: Commands,
+}
+
+#[derive(Subcommand)]
+/// The available commands.
+pub enum Commands {
+    #[command(about = "Run a simulation model.")]
+    /// Run a simulation model.
```

Comment:
With the doc comments, you don't actually need the `#[command(...)]` pragma above, e.g.:

```suggestion
    /// Run a simulation model.
```

I tried removing it and you can see the text for the doc comment in the program help. Kinda neat!

Reply:
I remember noticing this when I first implemented it and completely forgot about it, thanks.